packages/bff/.env

```diff
@@ -0,0 +1,6 @@
+HOST=0.0.0.0
+PORT=3100
```

Comment:
I suggest to use a different port number not already used by other services,
at the moment agreement and tenant process using 3100, so in case of local test It must be changed every time if agreement process is running.